api-specificatie/brc/openapi.yaml

```diff
@@ -0,0 +1,1129 @@
+    parameters:
+      - name: uuid
+        in: path
+        required: true
+        schema:
+          type: string
+          format: uuid
+  /besluitinformatieobjecten:
+    get:
+      operationId: besluitinformatieobject_list
+      description: >-
+        Geef een lijst van relaties tussen BESLUITen en INFORMATIEOBJECTen.
+
+
+        Door te filteren op besluit en/of informatieobject kan je opvragen in
+        welke
+
+        informatieobjecten een besluit is vastgelegd, of welke besluiten in een
+
+        informatieobject vastgelegd zijn.
+      parameters:
+        - name: besluit
+          in: query
+          description: URL to the related resource
+          required: false
+          schema:
+            type: string
+            format: uri
+        - name: informatieobject
+          in: query
+          description: >-
+            URL-referentie naar het informatieobject waarin (een deel van) het
+            besluit beschreven is.
+          required: false
+          schema:
+            type: string
+            format: uri
+      responses:
+        '200':
+          description: ''
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/BesluitInformatieObject'
+        '401':
+          description: ''
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Fout'
+        '403':
+          description: ''
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Fout'
+        '406':
+          description: ''
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Fout'
+        '409':
+          description: ''
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Fout'
+        '410':
+          description: ''
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Fout'
+        '415':
+          description: ''
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Fout'
+        '429':
+          description: ''
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Fout'
+        '500':
+          description: ''
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Fout'
+      tags:
+        - besluitinformatieobjecten
+    post:
+      operationId: besluitinformatieobject_create
+      description: |-
+        Registreer in welk(e) INFORMATIEOBJECT(en) een BESLUIT vastgelegd is.
+
+        Er wordt gevalideerd op:
+        - geldigheid besluit URL
+        - geldigheid informatieobject URL
```

Comment:
Laten we deze manier van documenteren mbt validatie ook vastleggen als ontwerp keuze.

Reply:
Toegevoegd!